bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/TextImpl.java

```diff
@@ -46,6 +46,13 @@ public class TextImpl extends AbstractFormComponentImpl implements Text {
     @SlingObject
     private Resource resource;
 
+    @Override
+    protected void initBaseModel() {
+        // Always make dataRef of Text component as null, for this we need to make unboundFormElement as true
+        unboundFormElement = Boolean.TRUE;
```

Comment:
Why do we need this override, when it is not exposed via edit dialog ?

Reply:
to make dataRef as null, otherwise it takes form hierarchy as its dataRef and resets on clicking reset